### PR TITLE
Streamline exercise without Finch

### PR DESCRIPTION
**Pull request by guilgaly**

### CODE REVIEW

**Review by slandelle — COMMENTED** [review]
So you decided to keep circe?

File: src/main/resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <!-- Configure log level here if needed. -->
+    <root level="info">
+        <appender-ref ref="STDOUT" />
+    </root>
+</configuration>
```
Comment (ccedric) [inline src/main/resources/logback.xml:11]: EOF

**Review by ccedric — COMMENTED** [review]
You should also add in the README that Java 11 is required.

wartremover checks are failing

**Comment by slandelle** [comment]
Why is Java 11 required?

**Comment by guilgaly** [comment]
I can probably easily remove the Java 11 requirement, I think it's just the method used for reading the file.

Another thing I just thought of: when describing the libraries used, we should mention that this uses version 2 and not 3 to avoid unnecessary confusion (the link to the doc is already correct).

**Comment by slandelle** [comment]
> I can probably easily remove the Java 11 requirement, I think it's just the method used for reading the file.

Just pull commons-io :)

**Comment by slandelle** [comment]
> Another thing I just thought of: when describing the libraries used, we should mention that this uses version 2 and not 3 to avoid unnecessary confusion (the link to the doc is already correct).

👍

**Pull request merged**